lib/ecto/adapters/sql.ex

```diff
@@ -60,6 +60,30 @@ defmodule Ecto.Adapters.SQL do
     * `:driver` (required) - the database driver library.
       For example: `:postgrex`
 
+  ## Debug log
+
+  You can get caller location when SQL called and log level is :debug.
```

Comment:
@josevalim 
I want to this log is default debug output, and get stacktrace location from `String.match?(/otp_app_name/)`.
But may not work well, so I created to users can set `log_stacktrace_match_path`.